dail/fmed-deduction.vbs

```diff
@@ -69,7 +70,8 @@ EndDialog
 
 Do
```

Comment:
The check_for_password needs to be in it's own do...loop. Add a do...loop for worker_signature